github-issue-classification/k8s/manifests/inference-deploy.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
```

Comment:
What would be the difference between this manifest and the [existing](https://github.com/intel/stacks-usecase/blob/master/github-issue-classification/k8s/manifests/inference-pod.yaml) one? Can we have make these changes there?

Reply:
this one is k8s deployment which also creates a k8s service exposed using NodePort. website server can access this service using nodeip:nodeport